Source/SourceKittenFramework/File.swift

```diff
@@ -10,12 +10,18 @@ import Foundation
 import SwiftXPC
 import SWXMLHash
 
+public typealias Line = (index: Int, content: String)
```

Comment:
this should probably go in `String+SourceKitten.swift` since it's generally applicable outside of `File`s.